src/main/java/org/cs1331/gitsubmitter/StudentSubmission.java

```diff
@@ -201,6 +201,7 @@ private Ref getHeadRef() throws Exception {
 
     public boolean pushFiles(String message, String ... fileNames)
             throws Exception {
+        message = message + "\n\nID: " + Utils.getIdentifier();
```

Comment:
Better to make a helper method:

    appendIdentifier(String message) {
        return "\n\nID: " + Utils.getIdentifier();
    }